tests/unit/cbprintf/testcase.yaml

```diff
@@ -1,209 +1,241 @@
 common:
   tags: cbprintf
   type: unit
-  filter: not CONFIG_PICOLIBC
```

Comment:
> tests/unit/cbprintf: Use minimal libc to test cbprintf code

should drop 202a8c4a86a4fbc0d9e9e487b0abea3af097b294

Reply:
done.